modules/tinymce/changelog.txt

```diff
@@ -1,3 +1,5 @@
+Version 5.7.0 (TBD)
+    Added console.error when advList has no List plugin as dependency #TINY-6585
```

Comment:
This isn't really a new feature, it's more a change of behaviour or a fix. So this should start with `Changed` or `Fixed` (probably changed). Additionally, `advList` isn't the plugin code, nor name so this should probably be ```advlist```. You also need to ensure names are in backticks.

So with all that in mind, maybe something like this?
```suggestion
    Changed the `advlist` plugin to log a console error message when the `list` plugin isn't enabled #TINY-6585
```

- [ ] Fix changelog wording and formatting

Reply:
Done.